### ISSUE TITLE
Bark notification delivery failed

### ISSUE THREAD

**Issue by jianyun8023** [louislam/uptime-kuma#2131]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

I set up bark notifications and clicking on the test will connect with an error. I called it in the docker container using `curl` and it was ok.
I see the address spliced in the log, the location of the bark api address is undefined.

<img width="933" alt="image" src="https://user-images.githubusercontent.com/18240360/192459212-27065d5a-dcf5-4876-9054-4cd1324355a2.png">

### 👟 Reproduction steps

Set a bark notification, save it, open it again, and click Test

### 👀 Expected behavior

Send notifications correctly

### 😓 Actual Behavior

Notification of failure to send

### 🐻 Uptime-Kuma Version

1.18.0

### 💻 Operating System and Arch

macOS 12

### 🌐 Browser

Chrome 105.0.5195.102

### 🐋 Docker Version

20.10.17

### 📝 Relevant log output

```shell
closeEmitted: false,
      [Symbol(kOnFinished)]: []
    },
    _events: [Object: null prototype] {
      response: [Function: handleResponse],
      error: [Function: handleRequestError],
      socket: [Function: handleRequestSocket]
    },
    _eventsCount: 3,
    _maxListeners: undefined,
    _options: {
      maxRedirects: 21,
      maxBodyLength: 10485760,
      protocol: 'http:',
      path: 'undefined&icon=https://github.com/louislam/uptime-kuma/raw/master/public/icon.png&group=UptimeKuma&sound=telegraph',
      method: 'GET',
      headers: [Object],
      agent: undefined,
      agents: [Object],
      auth: undefined,
      hostname: null,
      port: null,
      nativeProtocols: [Object],
      pathname: 'undefined&icon=https://github.com/louislam/uptime-kuma/raw/master/public/icon.png&group=UptimeKuma&sound=telegraph'
    },
    _ended: true,
    _ending: true,
    _redirectCount: 0,
    _redirects: [],
    _requestBodyLength: 0,
    _requestBodyBuffers: [],
    _onNativeResponse: [Function (anonymous)],
    _currentRequest: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      maxRequestsOnConnectionReached: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      _closed: false,
      socket: [Socket],
      _header: 'GET undefined&icon=https://github.com/louislam/uptime-kuma/raw/master/public/icon.png&group=UptimeKuma&sound=telegraph HTTP/1.1\r\n' +
        'Accept: application/json, text/plain, */*\r\n' +
        'User-Agent: axios/0.26.1\r\n' +
        'Host: localhost\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: nop],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: 'undefined&icon=https://github.com/louislam/uptime-kuma/raw/master/public/icon.png&group=UptimeKuma&sound=telegraph',
      _ended: false,
      res: null,
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'localhost',
      protocol: 'http:',
      _redirectable: [Circular *1],
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    _currentUrl: 'undefined&icon=https://github.com/louislam/uptime-kuma/raw/master/public/icon.png&group=UptimeKuma&sound=telegraph',
    [Symbol(kCapture)]: false
  },
  response: undefined,
  isAxiosError: true,
  toJSON: [Function: toJSON]

```
```

**Comment by louislam** [comment]
Should be fixed in the next release.

Related pull request: https://github.com/louislam/uptime-kuma/pull/2084

**Comment by jianyun8023** [comment]
@louislam Wow, that's awesome, thanks for your help!

**Comment by rezzorix** [comment]
Fixed in [1.18.1](https://github.com/louislam/uptime-kuma/releases/tag/1.18.1)
Can be closed here I assume?
